src/Component/Board/Board.spec.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import Board from './Board'
+import TestRenderer from 'react-test-renderer'
+
+describe('Component :: Board :: Board', () => {
+  it('loads when a card is creating', () => {
+    const inspector = TestRenderer.create(
+      <Board
+        isLoading={true}
+        cards={[]}
+      />
+    )
+
+    inspector.root.findByProps({ className: 'board loader' })
+  })
+
+  it('displays the cards and add "add another card button" when it is not creating any card', () => {
+    const inspector = TestRenderer.create(
+      <Board
+        isLoading={false}
+        cards={[{ title: 'My title' }]}
+      />
+    )
+
+    inspector.root.findByProps({ className: 'board add-card' })
+    inspector.root.findByProps({ className: 'card-title' })
+  })
```

Comment:
IMHO this tests belongs more to the container (index.js) as all the properties (or most of them) are coming from the store and never passed statically to it.
I wouldn't test the view here, I would make all the tests container level.
WDYT?

Reply:
Ok I will try to do this. 

I just adapted the tests from the `knp-cra`, I never used react-testing-library before.